### DISCUSSION TITLE
Recruiting ambassadors for FOK campaigns

### DISCUSSION THREAD

**Discussion by CooperSmout** [FreeOurKnowledge/website#42] [Ideas]
I think it could really help our campaigns to recruit 'ambassadors' who are keen to actively promote the campaign. Unfortunately, it's all too easy for people to sign a pledge and then forget about it, which makes it hard for our pledges to grow organically (I noticed this in the [Preregistration campaign](https://freeourknowledge.org/2020-12-03-preregistration-pledge/), which stalled after an initial upswing of activity). Ambassadors could really help here, by keeping the campaign fresh in people's minds and promoting it where possible. 

I'm not imagining this role to take a lot of time, but instead involve small tasks that could be completed semi-regularly (e.g. 15 mins/week). Hopefully people would do some of these things anyway, but I figured listing various ideas (and sharing strategies) could help us learn what works and develop more successful strategies in the future. In return for your efforts, we can show you off as an ambassador on the website and in our presentations, which could help with your visibility, and it might be another way to demonstrate your commitment to open science (e.g. listing in your resume). If we record people's efforts, we could also create a 'FOK leaderboard', which could be a fun way to reward ambassadors for their efforts :)

Here are some initial ideas (with estimated times):

_Once-off commitments_
* pre-sign the pledge before we go live (1 minute)
* write a personal twitter/facebook/linked-in post about why you signed the pledge (10 minutes)
* find and nominate 1+ person to play the role of '[champion](https://github.com/FreeOurKnowledge/website/discussions/39)' (30 minutes)
* attend the [workshop](https://github.com/FreeOurKnowledge/website/discussions/39) when we run it (1.5 hours)
* include a slide/mention in your presentations (e.g. show a [FOK badge](https://github.com/FreeOurKnowledge/website/blob/master/marketing/badge_preregistration.jpg) on the last slide) (5 min)
* insert a [FOK badge](https://github.com/FreeOurKnowledge/website/blob/master/marketing/badge_preregistration.jpg) on your posters (5 min)
* add your picture, affiliation and contact details to the FOK website (5 min)

_Weekly commitments_
* share each others' tweets/posts (5 minutes/week)
* help answer questions about the campaign if/when you can (5 min/week)
* comment/retweet 1+ relevant tweet/post and point people to the campaign (5 min/week)
* sign up 1 person per week?
* promote any pledgees who complete their pledge (e.g., share code) (5 mins/week)
* mention the campaign whenever it pops up/seems relevant in conversation :)

_Bonus commitments_
* help organise and/or run the [workshop](https://github.com/FreeOurKnowledge/website/discussions/39)
* help write the [FOK pledge publication](https://github.com/FreeOurKnowledge/website/discussions/37)
* add a reference/link to your homepage
* present a talk at your lab/institution about the campaign/FOK
* sign up other people to become ambassadors (super bonus points here!)
* ask guerilla questions at conferences (e.g. "as a leader in your field, have you taken any action or made pledges to [correct bias in scholarly publishing/improve openness/share code/etc]?")

**Comment by rmib200** [comment]
I'm all in. Can we recruit more ambassadors for the initiative in our local communities?

**Reply by CooperSmout** [reply]
Sure :) more the merrier I think! I guess the main thing will be to make sure that any new recruits are actually keen to put in some time on a regular basis. Do you think it's too much to ask ambassadors to record their efforts, e.g. in a spreadsheet/checklist? I was thinking we could maybe track people's contributions, so that we can reward them but also learn what works.

**Comment by I-Khormi** [comment]
I am happy to be an ambassador for FOK campaigns.

**Reply by CooperSmout** [reply]
Great! Let me know what you think of the tasks listed above, or if you have any other ideas for promoting.

**Comment by CooperSmout** [comment]
Hi everyone, I've created a form for ambassadors to complete [here](https://docs.google.com/forms/d/e/1FAIpQLScrvIalYnQh6L7MvrNdoc42LaDZ3nYujzLL0wudRjJBxWYXRw/viewform?usp=sf_link)

**Reply by I-Khormi** [reply]
> 
> 
> Hi everyone, I've created a form for ambassadors to complete [here](https://docs.google.com/forms/d/e/1FAIpQLScrvIalYnQh6L7MvrNdoc42LaDZ3nYujzLL0wudRjJBxWYXRw/viewform?usp=sf_link)

Done!

**Comment by meganEJcampbell** [comment]
Do people have good ideas for places to promote this ambassador role amongst less well represented groups?
